t/python/t0008-asyncio.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+
+###############################################################
+# Copyright 2022 Lawrence Livermore National Security, LLC
+# (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+#
+# This file is part of the Flux resource manager framework.
+# For details, see https://github.com/flux-framework.
+#
+# SPDX-License-Identifier: LGPL-3.0
+###############################################################
+
+import unittest
+
+import asyncio
+from flux.asyncio import loop
+import flux.job
+import flux.job.JobID
+import time
+
+def __flux_size():
+    return 1
+
+
+class TestFluxAsyncio(unittest.TestCase):
+    """
+    Tests for Flux Asyncio.
+    """
+
+    def test_as_completed(self):
```

Comment:
yeah I should have known that :)

I do think it makes sense to keep the sleep - the asyncio.sleep is using the timeout functionality, and if our implementation is wrong (or breaks) it could return before that is complete (which we'd want to know).